### ISSUE TITLE
Invalid syntax in save()

### ISSUE THREAD

**Issue by tmelot2** [tmelot2/folder-aliaser#3]
Specifically the dictionary unpacking on [line 58 of the plugin](https://github.com/tmelot2/folder-aliaser/blob/master/folder-aliaser.py#L58). The syntax being used was added in Python 3.5 and Sublime runs on 3.3.6.

From the console:
```
>>> import sys
>>> sys.version_info
sys.version_info(major=3, minor=3, micro=6, releaselevel='final', serial=0)
```

See [this comment in the PR](https://github.com/tmelot2/folder-aliaser/pull/2#discussion_r185589212) for more context. I saw [this thread](https://forum.sublimetext.com/t/upgrade-embedded-python-to-3-6/28883/3) where it is mentioned that they backport 3.4 patches to their own maintained 3.3.

**Comment by tedmiston** [comment]
Hmm, weird.  I'm trying to figure out how this didn't error out when I tested it on my Sublime.

**Comment by tedmiston** [comment]
Sorry about that!  Just sent you a PR to fix it #4.  Tested on my machine.

My hunch for missing this earlier is I must've had 2 versions installed or forgot to check out my branch when testing and ran the wrong one or something like that.

**Comment by tmelot2** [comment]
All good, thanks for the fix! I'm glad it happened, because I learned a little bit about Python in the process.
